Makefile

```diff
@@ -10,13 +10,21 @@ ifeq (,$(wildcard Makefile.global))
 endif
 
 include Makefile.global
-
+# directories with source files
+SUBDIRS +=  $(citus_top_builddir)/src/backend/columnar/safeclib
+ENSURE_SUBDIRS_EXIST := $(shell mkdir -p $(SUBDIRS))
```

Comment:
Should this be in `src/backend/columnar/Makefile` instead? It would seem more symmetrical with `src/backend/distributed/Makefile` and a slightly better place.

Reply:
but we do columnar "all" here in the top Makefile, and I got below error if I put it under columnar Makefile, " FATAL:  could not load library "/home/yanwjin/test/pg/install.13/lib/postgresql/citus.so": /home/yanwjin/test/pg/install.13/lib/postgresql/citus.so: undefined symbol: ColumnarTableSetOptions_hook",
 Run "ldd citus.so | grep columnar" also returns nothing

Reply:
done